dev/locations-page/locations.scss

```diff
@@ -19,7 +19,13 @@ $georgia: Georgia, serif !default;
 $gotham: "Gotham SSm A", "Gotham SSm B", Helvetica, Arial, sans-serif !default;
 $verdana: Verdana, Geneva, sans-serif;
 
+body {
```

Comment:
I'd recommend editing #contentcol instead of the global body element here, just to avoid modifying the site header/footer